tests/unit/sagemaker/image_uris/test_data_wrangler.py

```diff
@@ -60,23 +29,22 @@ def _test_ecr_uri(account, region, version):
     return expected_uri == actual_uri
 
 
-def test_data_wrangler_ecr_uri():
+@pytest.mark.parametrize("load_config", ["data-wrangler.json"], indirect=True)
+def test_data_wrangler_ecr_uri(load_config):
+    VERSIONS = extract_versions(load_config)
```

Comment:
please note above comment, this function does not give clarity that we are pulling processing versions.

Reply:
done